sim_appcenter_build_and_upload.sh

```diff
@@ -44,8 +44,9 @@ function create_sim_build() {
 
 function upload_sim_build() {
     local _build_path="$SIM_XCODE_DATA_PATH"/Build/Products/"$SIM_XCODE_CONFIGURATION"-iphonesimulator/"$SIM_XCODE_APP_NAME"
+    local _advanced_variables=${SIM_APPCENTER_ADVANCED_VARIABLES:-}
```

Comment:
Since the intent of this environment variable is to modify the behavior of the waldo upload command, I think a better name would be something like `SIM_WALDO_UPLOAD_OPTIONS`. Then you would have:
```
local _upload_options=${SIM_WALDO_UPLOAD_OPTIONS:-}
```